en/appendices/4-2-migration-guide.rst

```diff
@@ -41,6 +41,13 @@ Database
 - The ``TimeType`` will now correctly marshall values in the ``H:i`` format.
   Previously these values would be cast to ``null`` after validation.
   
+Error
+-----
+
+- ``ExceptionRenderer`` will now use the exception code as the HTTP status code
+for ``HttpException`` only. Other exceptions that should return a non-500
+HTTP code are controlled by ``ExceptionRenderer::$exceptionHttpCodes``.
```

Comment:
Should we also tell people how to update their application if they need to retain old behavior?

Reply:
Added note on how to override getHttpCode().